_proofs/sound/flxsound.md

```diff
@@ -0,0 +1,13 @@
+---
+title: FlxSound
+concept: Sound
+complexity: 1
+tags: [sound]
+order: 1
+---
+{% api flixel.FlxG %} provides a way of playing sounds, should you not feel like initiating a FlxSound instance.
+
+```haxe
+// Play a sound
+FlxG.sound.play();
```

Comment:
Might want to have some dummy argument in this example call, like `FlxG.sound.play(pathToSoundAsset);`.